VirusScanner.md

```diff
@@ -0,0 +1,53 @@
+#VirusScanner API
+
+&nbsp;
+
+#Contents
+
+- [Description](#description)
+- [Request](#request)
+- [Request Structure](#geocoding-request-structure)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+#Description
+
+This service runs advanced antivirus scans on any file type. The virus definitions that the service is based on are updated every five minutes to ensure that the service is as up to date as possible.
+
+The libraries that these virus scans are based on are being constantly updated by Avira. We recommend regular reflows of data to ensure that your data security is as up to date as possible. All files that return **is_safe = false** should be immediately quarantined and the source of the data should be investigated.
+
+----------------------------
+#Request
+
+A Virus Scanner request is simply the base64 string representation of a file to be scanned.
+
+#Request Structure
+
+| Parameter  | Description |
+|------------|-------------|
+| document     | Required. The base64 string representation of a file to be scanned. |
+
+&nbsp;
+
+----------------------
+#Response Structure
+
+All responses with an HTTP status of 200 will consist of a JSON object with a top-level "data" node containing the following elements:
+
+| Field    | Description |
+|----------|-------------|
+| is_safe  | A **boolean** value to quickly tell you whether or not the document is believed to contain a virus. |
+| virus_definition | A **string** value that will always be returned. If no virus is found, this will say "200 OK" - if a virus is found, this field will contain more specific information about the nature of the virus. |
```

Comment:
I can't remember exactly what it looks like off the top of my head, but the message will actually contain specific information about the type of virus found if that information is returned by Avira. For example, it could say something about it being a Word macro.